src/test/java/com/mongodb/hibernate/internal/translate/mongoast/filter/AstFieldOperationFilterTests.java

```diff
@@ -28,9 +27,7 @@ class AstFieldOperationFilterTests {
     @Test
     void testRendering() {
 
-        var astFilterFieldPath = new AstFilterFieldPath("fieldName");
-        var astComparisonFilterOperation = new AstComparisonFilterOperation(EQ, new AstLiteralValue(new BsonInt32(1)));
-        var astFieldOperationFilter = new AstFieldOperationFilter(astFilterFieldPath, astComparisonFilterOperation);
+        var astFieldOperationFilter = FilterTestUtils.createFieldOperationFilter("fieldName", EQ, new BsonInt32(1));
```

Comment:
`FilterTestUtils.createFieldOperationFilter` is statically imported in `AstLogicalFilterTests`, but not here. May we have those styles consistent?

Reply:
removed.